bsdploy/roles/jails_host/tasks/main.yml

```diff
@@ -121,7 +121,13 @@
   command: "tar -x -C /usr/jails/flavours/bsdploy_base --chroot --exclude '+*' -f /var/cache/pkg/All/pkg.txz creates=/usr/jails/flavours/bsdploy_base/usr/local/sbin/pkg"
 
 - name: The .ssh directory for root in bsdploy_base flavour
-  file: dest=/usr/jails/flavours/bsdploy_base/root/.ssh state=directory mode=0600 owner=root group=wheel
+  file:
+    dest: "/usr/jails/flavours/bsdploy_base/{{ploy_root_home_path}}/.ssh"
+    state: directory
+    mode: 0600
```

Comment:
Yaml is especially weird with numbers starting with 0. Search for yaml warts or gotchas.